panimg/image_builders/tiff.py

```diff
@@ -367,6 +393,14 @@ def _load_gc_files(
             if g.associated_files is not None
         ):
             gc_file = GrandChallengeTiffFile(file)
+            if file.suffix.lower() == ".dcm" and is_wsi_dicom(file.parent):
```

Comment:
It will fail for this tiff builder and raise an UnconsumedFilexException. It's tested here: https://github.com/DIAGNijmegen/rse-panimg/blob/1918fc9551f5edc9338954683804896bfbddc159/tests/test_wsi.py#L62

Reply:
What if the DICOM builder runs first? Does that correctly ignore WSI files?

Reply:
It does. I've added a test for it now.